package-lock.json

```diff
@@ -809,7 +809,7 @@
     },
     "@storybook/podda": {
       "version": "1.2.3",
-      "resolved": "https://registry.npmjs.org/@storybook/podda/-/podda-1.2.3.tgz",
+      "resolved": "http://registry.npmjs.org/@storybook/podda/-/podda-1.2.3.tgz",
```

Comment:
😱

Reply:
Removed the change to package-lock as it's not needed in this PR